ccpp/config/ccpp_prebuild_config.py

```diff
@@ -213,6 +213,10 @@
     'physics/physics/sfc_sice.f',
```

Comment:
Please change the `'physics/physics/mynnpbl_wrapper.F90',` line to `'physics/physics/mynnedmf_wrapper.F90',` to coincide with https://github.com/NCAR/ccpp-physics/pull/911/files#r860015386

Reply:
The meta files have the wrong name for two variables. I'm correcting that as well.